testutil/constructor/constructor.go

```diff
@@ -0,0 +1,41 @@
+// Package constructor provides constructors to easily initialize objects for test purpose with automatic error handling
+package constructor
+
+import (
+	"testing"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/stretchr/testify/require"
+
+	spntypes "github.com/tendermint/spn/pkg/types"
+)
+
+// Coins returns a sdk.Coins from a string
+func Coins(t testing.TB, str string) sdk.Coins {
```

Comment:
Is there any reason these (and some other of our testing functions) receive a `testing.TB` rather than a `*testing.T`?

My understanding is that the `TB` interface is for both testing and benchmarking, but I'm not sure how that functionality is being leveraged here.

Reply:
Well spotted, I added `TB` below as well so that the methods can be eventually used in Benchmarking test as well